environments/includes/ngrok.base.yml

```diff
@@ -0,0 +1,26 @@
+services:
+  ngrok:
+    hostname: "${WARDEN_ENV_NAME}-ngrok"
+    image: ngrok/ngrok:${NGROK_VERSION:-latest}
+    environment:
+      NGROK_AUTHTOKEN: ${NGROK_AUTHTOKEN}
+    volumes:
+      - ${WARDEN_ENV_PATH:-.}/.warden/ngrok.yml:/etc/ngrok.yml:ro,cached
+    command:
```

Comment:
@bruno-blackbird Reviewing the entrypoint script in the ngrok image, I think all you'd need to do here is just specify the `NGROK_CONFIG` environment variable to be `/etc/ngrok.yml`. The entrypoint script will start listening on all tunnels in the config file by default.

Reply:
@bap14 you mean instead of passing the command, yes, maybe it will be cleaner if I  add it this way

Reply:
@bruno-blackbird Yes, that's correct.